ipapython/certdb.py

```diff
@@ -578,7 +599,7 @@ def import_pkcs12(self, pkcs12_filename, pkcs12_passwd=None):
         try:
             self.run_pk12util(args)
         except ipautil.CalledProcessError as e:
-            if e.returncode == 17:
+            if e.returncode == 17 or e.returncode == 18:
```

Comment:
The import code (~L746) is calling `import_pkcs12` (this method), so don't we need to raise the new exceptions from this method also?

Reply:
Pff, I am probably tired at the end of the week... I intended to modify import_pkcs12, not export_pkcs12. Thanks for noticing my error.